ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ip-access-lists.j2

```diff
@@ -0,0 +1,130 @@
+{# eos - extended access-lists - improved data model #}
+{% if ip_access_lists is arista.avd.defined %}
+{#     set maximum number of ACL entries allowed, the playbook will fail if number is over the limit #}
+{%     set counter = namespace() %}
+{%     set counter.acle_number = 0 %}
+{#     START ACL CYCLE: walk over the list of defined ACLs #}
+{%     for acl in ip_access_lists %}
+{%         if not (acl.name is arista.avd.defined(fail_action='warning', var_name='ip_access_lists[*].name')) %}{# break cycle if not defined #}{% continue %}{% endif %}
```

Comment:
Sure. It's definitely not a problem to drop the warning and leave just arista.avd.defined(), but IMO it's better than failing silently. I know that it will be covered by schema, but I don't think it hearts to have the warning log right now. @carlbuchmann and @tgodaA let me know what do you think?

Reply:
I like same warnings in log files rather than too many warnings popped up when the playbook is running.

Reply:
ok. It will fail silently and we'll wait for schema.